apps/explorer/src/ui/stories/StatAmount.stories.tsx

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { type Meta, type StoryObj } from '@storybook/react';
+
+import { StatAmount, type StatAmountProps } from '../StatAmount';
+
+export default {
+    component: StatAmount,
+} as Meta;
+
+export const defaultAmount: StoryObj<StatAmountProps> = {
+    args: {
+        amount: 9740991,
+        currency: 'SUI',
+        dollarAmount: 123.56,
+        date: Date.now(),
```

Comment:
Not essential for now but you should lock dates in examples to a fixed time so that they don’t change over example times.